docs(adr): add ADR-001, SRD translation map, and data ingestion guide (#77)
cyrillrx/android-rpg: docs/i18n/data-ingestion.md

    "str": { "value": 8,  "savingThrowProficiency": "none" },
    "dex": { "value": 14, "savingThrowProficiency": "none" },
    "con": { "value": 10, "savingThrowProficiency": "none" },
    "int": { "value": 10, "savingThrowProficiency": "none" },
    "wis": { "value": 8,  "savingThrowProficiency": "none" },
    "cha": { "value": 8,  "savingThrowProficiency": "none" }
  },
  "skills": {
    "acrobatics": "none", "animalHandling": "none", "arcana": "none",
    "athletics": "none", "deception": "none", "history": "none",
    "insight": "none", "intimidation": "none", "investigation": "none",
    "medicine": "none", "nature": "none", "perception": "none",
    "performance": "none", "persuasion": "none", "religion": "none",
    "sleightOfHand": "none", "stealth": "proficient", "survival": "none"
  },
  "senses": { "darkvision": 18 },
  "damageAffinities": {},
  "conditionImmunities": {},
  "translations": {
    "fr": {
      "name": "Gobelin",
      "languages": ["commun", "gobelin"],
      "description": "<h2>Actions</h2><p>...</p>"
    }
  }
}
```

### Notes
- `abilities`: raw integer scores only; `savingThrowProficiency` is `"proficient"` or `"expert"` when applicable

> - `skills`: all 18 keys required; only non-`none` values need to be set

Author: ```suggestion
- `skills`: all 18 keys required (set to "none" if the creature is not proficient)
```
